crypto/dkg_feldmanvss.go

```diff
@@ -256,17 +267,23 @@ func (s *feldmanVSSstate) generateShares(seed []byte) error {
 	s.a = make([]scalar, s.threshold+1)
```

Comment:
It would be great to update the [godoc for this function](https://github.com/onflow/flow-go/pull/3421/files#diff-5ae00e788402d6f6773388800a459c091ef6e6ca6872c186de3ad0df1ec30e13R259), it looks unfinished.

Reply:
Can you please specify the function name, I couldn't find it with the link above

Reply:
Thanks! doc updated